android/src/main/java/com/reactnativepiwikprosdk/PiwikProSdkModule.kt

```diff
@@ -92,4 +114,14 @@ class PiwikProSdkModule(reactContext: ReactApplicationContext) : ReactContextBas
             promise.reject(error)
         }
     }
+
+    private fun cleanActionDimensions(tracker: Tracker) {
+      customDimensions.forEach {
+        if (it.value == "action") {
+          CustomDimension.setDimension(tracker.defaultTrackMe, it.key, null)
+        }
+
+        customDimensions.remove(it.key)
```

Comment:
Hmm, this seems to now clean some `visit` dimensions as well. Is this intentional?